pyscriptjs/src/components/pyrepl.ts

```diff
@@ -151,7 +151,7 @@ export function make_PyRepl(runtime: Runtime) {
         /** Execute the python code written in the editor, and automatically
          *  display() the last evaluated expression
          */
-        async execute(): Promise<void> {
+        execute(): null {
```

Comment:
Should the return type be `void` instead? 🤔

Reply:
I knew I should have gotten coffee before committing. Thanks Fabio!

Reply:
Haha I mean I guess void is kindof null? 😄